FHEM/00_SIGNALduino.pm

```diff
@@ -5793,9 +5796,11 @@ With a # at the beginnging whitelistIDs can be deactivated.
 	<ul>
 	<li>#define DEBUGMUREPEAT 1<br>
```

Comment:
der Teil gehört nich in die commandref vom Moudul finde ich.
Gerne ein Link ins Wiki oder in github. Was meinst Du?

Nebenbei arbeite ich daran, dass man sich die FW direkt aus github herunterladen kann. Aber gut, da hab ich noch Probleme mit meinem deploy script.

Reply:
Dann nehmen wir die 
<#define DEBUGMUREPEAT 1

wieder heraus ;-) kein Problem.
Ich werde mir am Nachmittag eine Übersicht verschaffen und dir dies gern neu "commiten".